projects/plugins/boost/app/assets/src/css/components/badge.scss

```diff
@@ -0,0 +1,11 @@
+.jb-badge {
+  border: 1px solid;
+  font-size: 0.5em;
+  text-transform: uppercase;
+  padding: 0.2em;
+  border-radius: 0.4em;
```

Comment:
Are we sure we need to add border-radius in em?

Reply:
I wanted to make the badges in a way that you can place them anywhere. By using em, I am making sure that the border radius reflects the font size.